README.md

```diff
@@ -11,54 +11,437 @@
+> [v2020-04-01](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-04-01) | [v0.8.15](https://github.com/input-output-hk/jormungandr/releases/tag/v0.8.15) | [1.9.3](https://github.com/input-output-hk/cardano-node/releases/tag/1.9.3)
+> [v2020-03-16](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2020-03-16) | [v0.8.14](https://github.com/input-output-hk/jormungandr/releases/tag/v0.8.14) | [1.6.0](https://github.com/input-output-hk/cardano-node/releases/tag/1.6.0)
 
-See [Wiki - Testing](https://github.com/input-output-hk/cardano-wallet/wiki/Testing)
 
 ## Documentation
+This readme includes some basic documentation to get you started, however, detailed documentation is available via the following links:
+
+Link                                                                                               | Audience
+-------------------------------------------------------------------------------------------------- | ------------------------------------------------------------
+[API Documentation](https://input-output-hk.github.io/cardano-wallet/api/edge)                     | Users of the Cardano Wallet API
+[Haddock Documentation](https://input-output-hk.github.io/cardano-wallet/haddock/edge)             | Haskell Developers using the `cardano-wallet` as a library
+[Wiki](https://github.com/input-output-hk/cardano-wallet/wiki)                                     | Anyone interested in the project and our development process
+[CLI Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-command-line-interface) | Users of the Cardano Wallet API
+
+--------------------------------------------------------------------------------
+
+### Rest Endpoint Summary
+
+- /wallets
+- /wallets/{walletId}
+- /wallets/{walletId}/tip
+- /wallets/{walletId}/statistics/utxos
+- /wallets/{walletId}/passphrase
+- /wallets/{walletId}/transactions
+- /wallets/{walletId}/payment-fees
+- /wallets/{walletId}/transactions/{transactionId}
+- /wallets/{walletId}/addresses
+- /stake-pools
+- /stake-pools/{stakePoolId}/wallets/{walletId}
+- /stake-pools/\*/wallets/{walletId}
+- /wallets/{walletId}/delegation-fees
+- /wallets/{walletId}/coin-selections/random
+- /byron-wallets
+- /byron-wallets/{walletId}
+- /byron-wallets/{walletId}/passphrase
+- /byron-wallets/{walletId}/tip
+- /byron-wallets/{walletId}/addresses
+- /byron-wallets/{walletId}/transactions
+- /byron-wallets/{walletId}/payment-fees
+- /byron-wallets/{walletId}/transactions/{transactionId}
+- /byron-wallets/{walletId}/migrations
+- /byron-wallets/{sourceWalletId}/migrations/{targetWalletId}
+- /byron-wallets/{walletId}/statistics/utxos
+- /network/information
+- /network/clock
+- /network/parameters/{epochId}
+- /proxy/transactions
```

Comment:
No. This is not maintainable and redundant with the API documentation side bar. We'll add endpoint, make changes etc ... Plus, having the list here doesn't really provide much information.